twisted/conch/ssh/keys.py

```diff
@@ -1121,6 +1191,11 @@ def _toString_AGENTV3(self):
             elif self.type() == 'DSA':
                 values = (data['p'], data['q'], data['g'], data['y'],
                           data['x'])
+            # I assume this is what's needed
+            # But commented out because it hasn't been tested yet.
+            elif self.type() == 'EC':
+                raise UnsupportedAlgorithm("toString() does not support Elliptic Curves yet.")
```

Comment:
I agree.  I was going to hold off on writing a ticket for this until the changes were accepted since it didn't make sense to have a ticket for something that's not in the baseline yet.

Once the PR goes through I'll put in a ticket.